set.mm

```diff
@@ -477711,6 +477837,26 @@ The sumset operation can be used for both group (additive) operations and
     $}
   $}
 
+  ${
+    idlsrgmulrss1.1 $e |- S = ( IDLsrg ` R ) $.
+    idlsrgmulrss1.2 $e |- B = ( LIdeal ` R ) $.
+    idlsrgmulrss1.3 $e |- .(x) = ( .r ` S ) $.
+    idlsrgmulrss1.4 $e |- .x. = ( .r ` R ) $.
+    idlsrgmulrss1.5 $e |- ( ph -> R e. CRing ) $.
+    idlsrgmulrss1.6 $e |- ( ph -> I e. B ) $.
+    idlsrgmulrss1.7 $e |- ( ph -> J e. B ) $.
+    $( In a commutative ring, the product of two ideals is a subset of the
```

Comment:
Of course it is also a subset of the second one as well and hence a subset of the intersection :-)

Reply:
Yes, I'll add a theorem about the intersection, that's an interesting one.